src/pages/Admin/Charity/EditProfile/index.tsx

```diff
@@ -37,6 +37,7 @@ function FormWithContext(props: EndowmentProfile) {
     kyc_donors_only: props.kyc_donors_only,
     overview: props.overview,
     url: props.url || "",
+    published: props.published || false,
```

Comment:
```suggestion
    published: props.published,
```
no need for fallback since this field is defined to be `boolean`, and is guaranteed to have value `false | true` 

on the otherdhand, if we are expecting `published` to be not present on all endowments - should update definition to
```ts
type EndowmentProfile = {
 //....
 publishable?:boolean
}
```